edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/deployment/KubernetesDeploymentMapper.cs

```diff
@@ -322,12 +323,18 @@ V1Volume GetVolume(KubernetesModule module, Mount mount)
             // Volume name will be customer defined name or modulename + mount.source
             if (this.persistentVolumeName.HasValue)
```

Comment:
if-else blocks is very confusing here. I suggest to make it a bit more easy to grasp:
```csharp
if (this.persistentVolumeName.HasValue)
{
    ...
   return new V1Volume(...);
}

if (this.storageClassName.HasValue)
{
    return new V1Volume(...);
}

return V1Volume(...);
```